app/controllers/unattended_controller.rb

```diff
@@ -1,5 +1,5 @@
 class UnattendedController < ApplicationController
-  include Foreman::Renderer
+  include UnattendedHelper # also includes Foreman::Renderer
```

Comment:
are we actually using it to begin with? I'm sure there are some dead methods in there?

Reply:
I have no clue, the whole rendering scope should not be the controller that renders the template imho.

Actually, I still have another issue with this:

```
# fresh foreman-rake console
UnattendedController.ancestors # does not include ForemanOmaha::RendererMethods
UnattendedController.send(:include, UnattendedHelper)
UnattendedController.ancestors # includes ForemanOmaha::RendererMethods now
```